<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>853</width>
    <height>630</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QListWidget" name="onlineList">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>20</y>
      <width>751</width>
      <height>191</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="fileMsg">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>230</y>
      <width>751</width>
      <height>271</height>
     </rect>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>540</y>
      <width>741</width>
      <height>31</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>853</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>开启</string>
    </property>
    <addaction name="actStart"/>
    <addaction name="actFileStart"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>关闭</string>
    </property>
    <addaction name="actClose"/>
    <addaction name="actFileClose"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actStart">
   <property name="text">
    <string>开启服务器</string>
   </property>
   <property name="toolTip">
    <string>开启服务器</string>
   </property>
  </action>
  <action name="actClose">
   <property name="text">
    <string>关闭服务器</string>
   </property>
   <property name="toolTip">
    <string>关闭服务器</string>
   </property>
  </action>
  <action name="actFileStart">
   <property name="text">
    <string>开启文件服务</string>
   </property>
   <property name="toolTip">
    <string>开启文件服务</string>
   </property>
  </action>
  <action name="actFileClose">
   <property name="text">
    <string>关闭文件服务</string>
   </property>
   <property name="toolTip">
    <string>关闭文件服务</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
